### PR TITLE
chunk: add clear in RowContainer

### PR DESCRIPTION
**Pull request by ichn-hu**
### What problem does this PR solve? 

This PR call `Clear` when we close a RowContainer.

### What is changed and how it works?

2 line of code to make it work.

### Check List 

Tests 

 - Unit test

### CODE REVIEW

**Comment by ichn-hu** [comment]
@SunRunAway @wshwsh12 PTAL, let's do a quick fix.

File: util/chunk/row_container.go [hunk 1]
```diff
@@ -171,6 +171,7 @@ func (c *RowContainer) Close() (err error) {
 		err = c.recordsInDisk.Close()
 		c.recordsInDisk = nil
 	}
+	c.records.Clear()
```
Comment (SunRunAway) [inline util/chunk/row_container.go]: Please add a unit test.

Comment (ichn-hu) [reply util/chunk/row_container.go]: Like what?

Comment (ichn-hu) [reply util/chunk/row_container.go]: Here we come, a simple ut.

**Comment by ichn-hu** [comment]
Slowly type a ?... Don't know why this PR breaks the CI.

**Comment by ichn-hu** [comment]
@SunRunAway @wshwsh12 PTAL, it now can be merged.

**Comment by ichn-hu** [comment]
@SunRunAway how about this?

**Pull request merged**